docs/counter_debug_desc.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Just use the old json in the README, because

1. it has comments
2. we don't need full long _asm_

Reply:
This is still NOT resolved.